scripts/train/train.py

```diff
@@ -197,7 +196,7 @@ def main(cfg: DictConfig) -> Trainer:
         cfg,
         TrainConfig,
         TRAIN_CONFIG_KEYS,
-        transforms=[update_batch_size_info],
+        transforms='all',
```

Comment:
for my own understanding, this will apply all transforms (according [to here](https://github.com/dakinggg/llm-foundry/blob/c16e94d740482f2d83424bf35de17714812e5d4f/llmfoundry/utils/config_utils.py#L253)) -- suppose we add a new transform that's only applicable for certain cases, would we still want to have `all` in this case? Would it be safer to only specify the transforms we currently want to use as a list in `train.py`, and only modify that list as we see fit down the line?

Reply:
resolved offline